system/ping/ping.c

```diff
@@ -131,21 +139,21 @@ static void ping_result(FAR const struct ping_result_s *result)
         break;
 
       case ICMP_E_SENDTO:
-        fprintf(stderr, "ERROR: sendto failed at seqno %u: %d\n",
+        fprintf(stderr, "ERROR: sendto failed at seqno %u: %ld\n",
```

Comment:
For: seqno use PRI macros for uint16_t and another non native types

Reply:
@davids5 we don't need apply PRI macro to uint16_t, since C standard require all type which's size is small than [unsigned]int must be promote to [unsigned]int for vararg function.
@wenliangxue please revert all seqno related change.

Reply:
all seqno remain unchanged